### DISCUSSION TITLE
Manually Installing https extension

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#3586] [General]
this code crash python
INSTALL 'httpfs';
LOAD 'httpfs';

is there a way to manually install the extension of windows 10 ? maybe it make sense to have https installed by default too ?

**Comment by Mause** [comment]
Can you paste the error you are getting?

**Reply by djouallah** [reply]
I don't know how to do it
![image](https://user-images.githubusercontent.com/12554469/167069573-c4bba6ae-fe21-47d0-8181-ad1793102af4.png)

**Comment by hannes** [comment]
This looks like a crash and should probably be investigated. @samansmink @pdet could you have a look? @djouallah, which exact version are you using?

**Comment by samansmink** [comment]
Sure!

An issue was created for this in https://github.com/duckdb/duckdb/issues/3587. 

@djouallah I answered your question on default installing httpfs there!
